src/views/Dissolutions/DefineDissolution.vue

```diff
@@ -95,7 +95,8 @@ import { RouteNames } from '@/enums'
 export default class DefineDissolution extends Mixins(CommonMixin, EntityFilterMixin, EnumMixin) {
   // Global getters
   @Getter getBusinessLegalName!: string
-  @Getter getCustodialRecordsResources!: any // TODO: Update to Custodial Resource IF
+  @Getter getCustodialRecordsResources!: CustodianResourceIF
```

Comment:
Still need this getter (and associated IF)?

Reply:
Yessir, just wanted to add the IF to the property. Forgot to do so previously.
This is used in the template.